README.md

```diff
@@ -72,6 +72,14 @@ info:     update {"description":"btc:usd","decimals":2,"roundID":"22","median":"
 info:     update {"description":"btc:usd","decimals":2,"roundID":"23","median":"5740350","updatedAt":"37820565","createdAt":"37820565"}
 ```
 
+# Request New Round
+
+Request a new round for a specific aggregator
+
+```
+yarn solink request-round <aggregator-id>
```

Comment:
is aggregator-id the public key or a name like `btc:usd`?

Reply:
is the pair name like `btc:usd`, I will add an example